firefox-ios/Client/Frontend/Browser/Tabs/Legacy/LegacyInactiveTabs/LegacyInactiveTabViewModel.swift

```diff
@@ -48,46 +48,21 @@ class LegacyInactiveTabViewModel {
         updateFilteredTabs()
     }
 
+    // FIXME What is this comment saying? I don't have context... the dates described are wrong, and it didn't use the
+    // debug test wrapper like the 2 other places this logic was used.
+    //
     /// This function returns any tabs that are less than four days old.
     ///
     /// Because the "Jump Back In" and "Inactive Tabs" features are separate features,
     /// it is not a given that a tab has an active/inactive state. Thus, we must
     /// assume that if we want to use active/inactive state, we can do so without
     /// that particular feature being active but still respecting that logic.
-    static func getActiveEligibleTabsFrom(_ tabs: [Tab], profile: Profile) -> [Tab] {
-        var activeTabs = [Tab]()
-
-        let currentDate = Date()
-        let noon = Calendar.current.date(bySettingHour: 12, minute: 0, second: 0, of: currentDate) ?? Date()
-        let day14Old = Calendar.current.date(byAdding: .day, value: -14, to: noon) ?? Date()
-        let defaultOldDay = day14Old
-
-        for tab in tabs {
-            let tabTimeStamp = tab.lastExecutedTime ?? tab.firstCreatedTime ?? 0
-            let tabDate = Date.fromTimestamp(tabTimeStamp)
-
-            if tabDate > defaultOldDay || tabTimeStamp == 0 {
-                activeTabs.append(tab)
-            }
-        }
-
-        return activeTabs
```

Comment:
@ih-codes Thanks for all of this work. 👏 We actually had an old ticket to refactor this so I went ahead and closed this one also: https://mozilla-hub.atlassian.net/browse/FXIOS-9188

Reply:
Thanks @mattreaganmozilla!! Little did I know. 😂